src/modules/services_manager_service.rb

```diff
@@ -47,16 +45,14 @@ def read
         load_units
 
         @supported_unit_files = unit_files.select do |_, status|
-          Status::SUPPORTED_STATES.member?(status)
+          status != Status::MASKED # masked services should not been shown at all
```

Comment:
Ok, great. I was asking if you should remove the `Status::SUPPORTED_STATES` constant (as AFAIK it's not used anymore).